src/Input.test.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import {
+  fireEvent, render,
+} from '@testing-library/react';
+
+import Input from './Input';
+
+describe('Input component Test', () => {
+  describe('1. input 컴포넌트 render 할 때', () => {
+    test('label,input,button 출력되어야 함 ', () => {
+      const onChange = jest.fn();
+      const { getByText, getByPlaceholderText, getByRole } = render(
+        <Input
+          value="할일!"
+          onChange={onChange}
+        />,
+      );
```

Comment:
```suggestion
      const { getByText, getByPlaceholderText, getByRole } = render((
        <Input
          value="할일!"
          onChange={onChange}
        />
      ));
```
괄호를 하나씩 더 붙여주시면 `,` 를 없앨 수 잇어요 :)